pom.xml

```diff
@@ -40,6 +40,30 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.json/json -->
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20200518</version>
+		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.dataformat</groupId>
+			<artifactId>jackson-dataformat-yaml</artifactId>
+			<version>2.9.9</version>
+		</dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.6.2</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Please format the indent here.

Reply:
It is already indented.Is there any different way to indent ?

Reply:
The indent for this `dependency` is different than the others.  I think your editor might be able to auto-fix this.

Reply:
Oh I just noticed it shows up fine in my IDE.I will have a look